cmd/config.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/pkg/errors"
+	"github.com/redhat-developer/odo/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+// configurationCmd represents the app command
+var configurationCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Modifies configuration settings",
+	Long: `Modifies Odo specific configuration settings within the config file.
+   # Available Parameters:
+     UpdateNotification - Controls if an update notification is shown or not (true or false)`,
+	Example: fmt.Sprintf("%s\n%s\n",
+		configurationViewCmd.Example,
+		configurationSetCmd.Example),
+	Aliases: []string{"configuration"},
+	// 'odo utils config' is the same as 'odo utils config --help'
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) >= 1 && args[0] != "view" && args[0] != "set" {
+			return fmt.Errorf("Unknown command, use set or view")
+		}
+		return nil
+	}, Run: func(cmd *cobra.Command, args []string) {
+		if len(args) > 0 && args[0] == "set" {
+			configurationSetCmd.Run(cmd, args)
+		} else if len(args) > 0 && args[0] == "view" {
+			configurationViewCmd.Run(cmd, args)
+		} else {
+			cmd.Help()
+		}
+	},
+}
+
+var configurationSetCmd = &cobra.Command{
+	Use:   "set",
+	Short: "Set a value in odo config file",
+	Long: `Set an individual value in the Odo configuration file 
+   # Available Parameters:
+     UpdateNotification - Controls if an update notification is shown or not (true or false)`,
+	Example: `  # Set a configuration value
+    odo utils config set UpdateNotification false`,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 2 {
+			return fmt.Errorf("Please provide a parameter name and value")
+		} else if len(args) > 2 {
+			return fmt.Errorf("Only one value per parameter is allowed")
+		} else {
+			return nil
+		}
+	}, RunE: func(cmd *cobra.Command, args []string) error {
+		cfg, err := config.New()
+		if err != nil {
+			return errors.Wrapf(err, "unable to set configuration")
+		}
+		value, err := strconv.ParseBool(args[1])
+		if err != nil {
+			return errors.Wrapf(err, "unable to set configuration")
+		}
+		return cfg.SetConfiguration(strings.ToLower(args[0]), value)
+	},
+}
+
+var configurationViewCmd = &cobra.Command{
+	Use:   "view",
+	Short: "View current values",
+	Long:  "View current configuration",
+	Example: `  # For viewing the current configuration
+    odo utils config view`,
```

Comment:
need to add a newline (see above). also change the two tabs to 3-spaces instead (it's messing up alignment when using `odo utils --help`